docs/en_us/internal/testing.rst

```diff
@@ -465,6 +465,48 @@ relative to the ``common/test/acceptance/tests`` directory. This is an example f
 
     paver test_bokchoy --extra_args="-a 'a11y'" -t test_lms_dashboard.py:LmsDashboardA11yTest.test_dashboard_course_listings_a11y
 
+Options for Faster Development Cycles in Bok-Choy Tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following are ways in which a developer could shorten the development
+cycle for faster feedback. The options below can often be used together.
+
+**Multiprocessing Mode**
+Bok-choy tests can be threaded using the `-n` switch.  Using 2 threads generally
+reduces test cycles by 33%.  The recommendation is to make sure the
+number of threads is no more than the number of processors available. For
+example, the Cypress release of devstack is provisioned by default with 2
+processors. In that case, to run tests in multiprocess mode::
+
+    paver test_bokchoy -n 2
+
+*Caveat*: Not all tests have been designed with multiprocessing in mind; some
+testcases will fail in multiprocess mode for various reasons (e.g., shared
```

Comment:
Is there any way to tell which tests these are?  I could see it potentially taking a dev a while (more than the 33% time savings) to figure out that the reason a test is failing is because of the multiprocessing.

Are there common ways that these would fail?